src/main/java/de/hglabor/worldfeatures/features/util/TemperaturesFeature.java

```diff
@@ -67,189 +147,303 @@ public void onServerStart(Plugin plugin) {
-                                    player.getLocation().clone().add(-1,0,-1).getBlock().setType(Material.BLUE_ICE);
-                                    player.getLocation().clone().add(1,0,-1).getBlock().setType(Material.BLUE_ICE);
-                                    player.getLocation().clone().add(-1,0,1).getBlock().setType(Material.BLUE_ICE);
-                                }
-                                if(getTemperatureLevel(player) < -29) {
-                                    player.getLocation().clone().add(0,1,0).getBlock().setType(Material.BLUE_ICE);
-                                    player.getLocation().clone().add(1,1,0).getBlock().setType(Material.BLUE_ICE);
-                                    player.getLocation().clone().add(0,1,1).getBlock().setType(Material.BLUE_ICE);
-                                    player.getLocation().clone().add(1,1,1).getBlock().setType(Material.BLUE_ICE);
-                                    player.getLocation().clone().add(0,-1,0).getBlock().setType(Material.BLUE_ICE);
-                                    player.getLocation().clone().add(-1,1,0).getBlock().setType(Material.BLUE_ICE);
-                                    player.getLocation().clone().add(0,1,-1).getBlock().setType(Material.BLUE_ICE);
-                                    player.getLocation().clone().add(-1,1,-1).getBlock().setType(Material.BLUE_ICE);
-                                    player.getLocation().clone().add(1,1,-1).getBlock().setType(Material.BLUE_ICE);
-                                    player.getLocation().clone().add(-1,1,1).getBlock().setType(Material.BLUE_ICE);
-                                    player.getLocation().clone().add(0,0,0).getBlock().setType(Material.BLUE_ICE);
-                                    player.getLocation().clone().add(1,0,0).getBlock().setType(Material.BLUE_ICE);
-                                    player.getLocation().clone().add(0,0,1).getBlock().setType(Material.BLUE_ICE);
-                                    player.getLocation().clone().add(1,0,1).getBlock().setType(Material.BLUE_ICE);
-                                    player.getLocation().clone().add(0,-0,0).getBlock().setType(Material.BLUE_ICE);
-                                    player.getLocation().clone().add(-1,0,0).getBlock().setType(Material.BLUE_ICE);
-                                    player.getLocation().clone().add(0,0,-1).getBlock().setType(Material.BLUE_ICE);
-                                    player.getLocation().clone().add(-1,0,-1).getBlock().setType(Material.BLUE_ICE);
-                                    player.getLocation().clone().add(1,0,-1).getBlock().setType(Material.BLUE_ICE);
-                                    player.getLocation().clone().add(-1,0,1).getBlock().setType(Material.BLUE_ICE);
-                                    player.damage(new Random().nextInt(40));
+                        for(Block block1 : new GasFeature().getNearbyBlocks(player.getLocation(), 3)) {
```

Comment:
KEINE NEUEN FEATURE INSTANCES ERZEUGEN!!! mach die methode static am besten oder in eine Utils class